packages/arb-ts/scripts/deployStandard.ts

```diff
@@ -0,0 +1,131 @@
+import { instantiateBridge } from './instantiate_bridge'
+import { ERC20__factory } from '../src/lib/abi/factories/ERC20__factory'
+import dotenv from 'dotenv'
+import args from './getCLargs'
+import { constants, BigNumber, utils } from 'ethers'
+
+dotenv.config()
+
+const privKey = process.env.PRIVKEY as string
+if (!privKey) {
+  throw new Error('Set PRIVKEY env variable')
+}
+
+if (!args.l1_token_address) {
+  throw new Error(
+    'Include l1_token_address arg (--l1_token_address 0xmytokenaddress)'
+  )
+}
+if (!args.amount) {
+  throw new Error('Include amount arg (--amount 123)')
+}
+
+const { l1_token_address: l1TokenAddress, amount } = args
+const amountBigNum = BigNumber.from(amount)
+
+const main = async () => {
+  const { bridge } = await instantiateBridge(privKey, privKey)
+  const walletAddress = await bridge.l1Bridge.getWalletAddress()
+
+  const l1Token = ERC20__factory.connect(l1TokenAddress, bridge.l1Provider)
+  // TODO: check blacklist
+
+  /* Looks like an L1 token: */
+  try {
+    await l1Token.symbol()
+    await l1Token.name()
+    await l1Token.decimals()
+    await l1Token.balanceOf(walletAddress)
+  } catch (err) {
+    console.warn(`${l1TokenAddress} doesn't look like an L1 ERC20 token`)
+    throw err
+  }
+  /* check that you have some eth */
+  const walletBal = await bridge.l1Provider.getBalance(walletAddress)
+  if (walletBal.eq(constants.Zero)) {
+    throw new Error(`${walletAddress} has no Ether to pay for gas`)
+  }
+  /* check token bal */
+  const tokenBal = await l1Token.balanceOf(walletAddress)
```

Comment:
we already check `await l1Token.balanceOf(walletAddress)` we can keep that value instead of querying again